cmd/podman/images.go

```diff
@@ -159,6 +159,17 @@ func imagesCmd(c *cliconfig.ImagesValues) error {
 		filters = append(filters, fmt.Sprintf("reference=%s", image))
 	}
 
+	var sortValues = map[string]bool{
+		"id":         true,
+		"size":       true,
+		"tag":        true,
+		"repository": true,
+		"created":    true,
+	}
+	if !sortValues[c.Sort] {
+		return errors.Errorf("invalid sort value %q", c.Sort)
```

Comment:
We could print valid values in the message.

Reply:
Fixed